RIPS/rip-7560.md

```diff
@@ -811,6 +611,14 @@ the new transaction type.
 [EIP-3607](https://eips.ethereum.org/EIPS/eip-3607) introduces a ban on transactions from senders with deployed code.
 This limitation does not apply to AA_TX_TYPE transactions.
 
+### Deterministic addresses and Smart Contract Account `nonce` value
+
+Before RIP-7560, for accounts with associated code (smart contracts), the account nonce is only used and incremented
+when the account executes the `CREATE` (`0xf0`) opcode.
+
+With RIP-7560, the `sender` account's `nonce` will be incremented before a transaction execution starts.
+Developer tooling that rely on counterfactual addresses should take that into account or risk missing the right address.
```

Comment:
That's only an issue with CREATE though, not CREATE2, right?

Reply:
Right. Added an explanation.